modules/caddyhttp/templates/tplcontext.go

```diff
@@ -398,6 +401,34 @@ func (c TemplateContext) funcHTTPError(statusCode int) (bool, error) {
 	return false, caddyhttp.Error(statusCode, nil)
 }
 
+// formatHuman transforms size and time inputs to a human readable format.
+// For time formatting, the default format is RFC1123Z.
+// The supported time formats: https://pkg.go.dev/time#pkg-constants
+func (c TemplateContext) formatHuman(function, data string) (string, error) {
+
+	switch function {
+	case "size":
+
+		dataint, dataerr := strconv.ParseUint(data, 10, 64)
+		if dataerr != nil {
+			return "", fmt.Errorf("size: data can't be parsed. Error: %s", dataerr.Error())
+		}
+		return humanize.Bytes(dataint), nil
+	case "time":
+
+		// TODO: add optional layout argument
+		//timelayout := "Mon Jan 02 2006 15:04:05 GMT-0700"
+		dataint, dataerr := time.Parse(time.RFC1123Z, data)
```

Comment:
I'm thinking maybe it could be done by taking the `function` input like `time:Mon Jan 02 2006 15:04:05 GMT-0700`, so like `time:` being a prefix to say which kind of format, then the part after being the format/layout for that kind 🤔